automated-demo/auto-demo.sh

```diff
@@ -199,11 +199,11 @@ check_federated_clusters_ready()
   WAIT=0
   MAX_WAIT=300
   echo "Checking if Federated Clusters are ready"
-  FEDERATED_CLUSTERS_STATE_READY=$(oc --context=feddemocl1 describe kubefedclusters -n $DEMO_NAMESPACE | grep -c ClusterReady)
+  FEDERATED_CLUSTERS_STATE_READY=$(oc --context=feddemocl1 describe kubefedclusters -n kube-federation-system | grep -c ClusterReady)
```

Comment:
I'd suggest creating a new GLOBAL VAR for storing the value of "kube-federation-system", so if that name changes in the future we just need to update 1 var value and 0 lines of code.